From caf3738e97cc1aed5b256c0dc93f4b4265f6b8fa Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Tue, 3 Mar 2020 10:24:56 +0700
Subject: [PATCH 008/197] arm64: dts:renesas: r8a774c0: Add the emergency node
 in thermal-zones

This patch adds the emergency node in thermal zones to support
for R8A774C0 SoC.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774c0.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774c0.dtsi b/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
index 2b7b1128..c79848a 100644
--- a/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
@@ -1921,6 +1921,14 @@
 	};
 
 	thermal-zones {
+		emergency {
+			polling-delay = <1000>;
+			on-temperature = <110000>;
+			off-temperature = <95000>;
+			target_cpus = <&a53_1>;
+			status = "disabled";
+		};
+
 		cpu-thermal {
 			polling-delay-passive = <250>;
 			polling-delay = <0>;
-- 
2.7.4

